src/main/java/blackjack/domain/BlackJackGame.java

```diff
@@ -0,0 +1,97 @@
+package blackjack.domain;
+
+import blackjack.domain.dto.GameResultDto;
+import blackjack.domain.dto.ParticipantDto;
+import blackjack.domain.strategy.RandomCardGenerator;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class BlackJackGame {
+    private final List<Player> players;
+    private final Dealer dealer;
+    private final CardDeck cardDeck;
+
+    public BlackJackGame(List<String> playersNames) {
+        this.cardDeck = new CardDeck(new RandomCardGenerator());
+        this.dealer = new Dealer(cardDeck.drawTwoCards());
+        validateEmptyNames(playersNames);
+        this.players = createPlayers(playersNames);
+    }
+
+    private void validateEmptyNames(List<String> playerNames) {
+        if (playerNames.isEmpty()) {
+            throw new IllegalArgumentException("[ERROR] 빈 이름이 있습니다.");
+        }
+    }
+
+    private List<Player> createPlayers(List<String> playersNames) {
+        return playersNames.stream()
+                .map(playerName -> new Player(playerName.trim(), cardDeck.drawTwoCards()))
+                .collect(Collectors.toList());
+    }
+
+    public List<ParticipantDto> getParticipantsDto() {
+        List<ParticipantDto> participantDtos = new ArrayList<>();
+        participantDtos.add(ParticipantDto.of(dealer, List.of(dealer.showFirstCard())));
+        for (Participant player : players) {
+            participantDtos.add(ParticipantDto.of(player, player.showCards()));
+        }
+        return participantDtos;
+    }
+
+    public List<ParticipantDto> getFinalParticipantsDto() {
+        List<ParticipantDto> participantDtos = new ArrayList<>();
+        participantDtos.add(ParticipantDto.of(dealer, dealer.showCards()));
+        for (Participant player : players) {
+            participantDtos.add(ParticipantDto.of(player, player.showCards()));
+        }
+        return participantDtos;
+    }
+
+    public boolean isAnyPlayerTurnRemain() {
+        return players.stream()
+                .anyMatch(participant -> !participant.isFinished());
+    }
+
+    public String playNameOnTurn() {
+        return players.stream()
+                .filter(participant -> !participant.isFinished())
+                .findFirst().get()
+                .getName();
+    }
+
+    public List<Card> playerDrawCardOnDecision(Boolean decision) {
```

Comment:
```suggestion
    public List<Card> playerDrawCardOnDecision(boolean decision) {
```
여긴 레퍼런스타입으로 받으신 이유가 있을까요?

Reply:
부끄럽지만 Boolean / boolean 차이를 모르고 있다가 리뷰를 보고 찾아보게 되었어요 😅 감사합니다!